projects/mono.md

```diff
@@ -0,0 +1,31 @@
+# Mono
+
+Mono, the open source development platform based on the .NET Framework, allows developers to build cross-platform applications with improved developer productivity. Mono’s .NET implementation is based on the ECMA standards for [C#](http://www.ecma-international.org/publications/standards/Ecma-334.htm) and the [Common Language Infrastructure](http://www.ecma-international.org/publications/standards/Ecma-335.htm).
```

Comment:
Thanks - just removed that section as it was duplication of the text below as well as being incorrect.